official/projects/cots_detector/crown_of_thorns_starfish_detection_pipeline.ipynb

```diff
@@ -64,7 +64,7 @@
     "id": "jDiIX2xawkJw"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        updated_tracks.append(track.replace(det=det))
</pre><p>You're calling track.replace(det) twice.  And it looks like updated_tracks isn't used anywhere else?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/models/pull/10640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>